fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/FedoraTypes.java

```diff
@@ -41,6 +41,8 @@
 
     String FEDORA_TOMBSTONE = "fedora:Tombstone";
 
+    String FEDORA_TIME_MAP = "fedora:TimeMap";
```

Comment:
We should have a follow-on ticket to add `TimeMap` to the [Fedora ontology](https://github.com/fcrepo4/ontology/blob/master/repository.rdf).